framework/db/sqlite/QueryBuilder.php

```diff
@@ -141,16 +141,13 @@ public function resetSequence($tableName, $value = null)
                 $key = reset($table->primaryKey);
                 $tableName = $db->quoteTableName($tableName);
                 $value = $this->db->useMaster(function (Connection $db) use ($key, $tableName) {
-                    return $db->createCommand("SELECT MAX($key) FROM $tableName")->queryScalar();
+                    return $db->createCommand("SELECT MAX({$key}) FROM {$tableName}")->queryScalar();
```

Comment:
@sergeymakinen If `{}` is unnecessary, why do you want to use it? Lets keep CS simple and consistent - it is easier to avoid using unnecessary `{}` than add them wherever possible.